WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -697,6 +699,28 @@ - (CGFloat)heightForSearchWrapperView
 
 #pragma mark - Filter related
 
+- (BOOL)canFilterByAuthor
+{
+    return [self.blog isMultiAuthor] && self.blog.account.userID && [self.blog isHostedAtWPcom];
+}
+
+- (BOOL)shouldShowOnlyMyPosts
+{
+    PostAuthorFilter filter = [self currentPostAuthorFilter];
+    return filter == PostAuthorFilterMine;
+}
+
+- (PostAuthorFilter)currentPostAuthorFilter
+{
+    return PostAuthorFilterEveryone;
+}
+
+- (void)setCurrentPostAuthorFilter:(PostAuthorFilter)filter
+{
+    // Noop. The default implementation is read only.
+    // Subclasses may override the getter and setter for their own filter storage.
```

Comment:
Strictly speaking, it doesn't expose an instance variable, so not really an accessor :stuck_out_tongue: 

It doesn't bother me too much though, I was just being pedantic here